cupyx/__init__.py

```diff
@@ -4,6 +4,7 @@
 from cupyx.scatter import scatter_add  # NOQA
 from cupyx.scatter import scatter_max  # NOQA
 from cupyx.scatter import scatter_min  # NOQA
+from cupyx.time import run  # NOQA
```

Comment:
I think the name `cupyx.run` is too generic; I guess most users cannot imagine the feature of this function from the name.

Reply:
OK. I will remove this line and let users to call it `cupyx.time.run` explicitly.

Reply:
BTW, `cupy.time.repeat` seems better... I will rename it.

Reply:
Fixed.